multiaccounts/accounts/database.go

```diff
@@ -559,6 +560,9 @@ func (db *Database) GetSettings() (Settings, error) {
 		&s.AutoMessageEnabled,
 	)
 
+  communitiesSettings, err := db.GetCommunitiesSettings()
+  s.CommunitiesSettings = communitiesSettings
```

Comment:
Since `communities_settings` live in a separate table, I'm explicitly querying/assigning it here.

Let me know if this should not happen here and whether we prefer a separate RPC API so clients can request that data.